sdk/identity/Azure.Identity/src/Credentials/AzurePipelinesCredential.cs

```diff
@@ -33,29 +34,34 @@ protected AzurePipelinesCredential()
         /// <summary>
         /// Creates a new instance of the <see cref="AzurePipelinesCredential"/>.
         /// </summary>
-        /// <param name="tenantId">The tenant ID for the service connection.</param>
+        /// <param name="systemAccessToken">The pipeline's <see href="https://learn.microsoft.com/azure/devops/pipelines/build/variables?view=azure-devops%26tabs=yaml#systemaccesstoken">System.AccessToken</see> value.</param>
         /// <param name="clientId">The client ID for the service connection.</param>
-        /// <param name="serviceConnectionId">The service connection ID, as found in the querystring's resourceId key.</param>
+        /// <param name="tenantId">The tenant ID for the service connection.</param>
+        /// <param name="serviceConnectionId">The service connection Id for the service connection associated with the pipeline.</param>
         /// <param name="options">An instance of <see cref="AzurePipelinesCredentialOptions"/>.</param>
-        /// <exception cref="System.ArgumentNullException">When <paramref name="tenantId"/>, <paramref name="clientId"/>, or <paramref name="serviceConnectionId"/> is null.</exception>
-        public AzurePipelinesCredential(string tenantId, string clientId, string serviceConnectionId, AzurePipelinesCredentialOptions options = default)
+        /// <exception cref="ArgumentNullException">When <paramref name="systemAccessToken"/> is null.</exception>
+        public AzurePipelinesCredential(string systemAccessToken, string clientId, string tenantId, string serviceConnectionId, AzurePipelinesCredentialOptions options = default)
```

Comment:
Constructor args are generally tenant, client, ...
```suggestion
        public AzurePipelinesCredential(string tenantId, string clientId, string serviceConnectionId, string systemAccessToken, AzurePipelinesCredentialOptions options = default)
```